.github/workflows/bot.yml

```diff
@@ -58,17 +58,29 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Cache github-bot binary
+        id: cache-bot-bin
+        uses: actions/cache@v4
+        with:
+          path: /usr/local/bin/github-bot
+          key: ${{ runner.os }}-github-bot-${{ hashFiles('contribs/github-bot/**/*', '!contribs/github-bot/**/*.md', '!contribs/github-bot/Makefile') }}
```

Comment:
```
      - name: Get last commit hash for directory
        id: get-hash
        run: echo "dir_hash=$(git log -1 --format="%H" -- contribs/github-bot/)" >> $GITHUB_OUTPUT

      - name: Cache github-bot binary
        uses: actions/cache@v3
        with:
          path: path/to/cache
          key: ${{ runner.os }}-deps-${{ steps.get-hash.outputs.dir_hash }}
```

What about something like this?

Reply:
This version does not ignore the changes made to the `Makefile` or the `README.md` (which can be safely ignored), right? 
Why would you prefer this version?

Reply:
You're right, yes.

The reason why I prefer it is that it seems more straightforward to debug: you can go back to the commit hash from the cache key directly.

Reply:
Ok done: https://github.com/gnolang/gno/pull/4146/commits/5b68d8289c12faf3da2b17fc4a81c678d72408cc